java/ql/lib/semmle/code/java/frameworks/Strings.qll

```diff
@@ -64,7 +64,14 @@ private class StringSummaryCsv extends SummaryModelCsv {
         "java.lang;StringBuilder;true;StringBuilder;;;Argument[0];Argument[-1];taint;manual",
         "java.lang;CharSequence;true;charAt;;;Argument[-1];ReturnValue;taint;manual",
         "java.lang;CharSequence;true;subSequence;;;Argument[-1];ReturnValue;taint;manual",
-        "java.lang;CharSequence;true;toString;;;Argument[-1];ReturnValue;taint;manual"
+        "java.lang;CharSequence;true;toString;;;Argument[-1];ReturnValue;taint;manual",
+        "java.util;StringJoiner;true;StringJoiner;;;Argument[0..2];Argument[-1];taint;manual",
```

Comment:
Not sure if it matters, but I think these could all be `false` because `StringJoiner` is `final`.
```suggestion
        "java.util;StringJoiner;false;StringJoiner;;;Argument[0..2];Argument[-1];taint;manual",
```

Reply:
@atorralba, not sure if this is relevant, but the tests still have comments with <code>java.util;StringJoiner;<b>true</b></code>.

Reply:
Well, it's better that we stay accurate 😄 Fixed, thanks!